validator/packaging/validator.toml.example

```diff
@@ -0,0 +1,17 @@
+#
+# Sawtooth Lake -- Validator Configuration
+#
+
+# This file should exist in the defined config directory and allows
+# validators to be configured without the need for command line options.
+# It is important to note that public_uri must be set by either the config or
+# the command line.
+
+# The following is a possible example.
+
+network_endpoint = "tcp://127.0.0.1:8800"
```

Comment:
Aren't they already documented in the validator CLI?. Can we instead reference the cli, explain how to go between the two, and remove the comments from this file so there isn't duplicate documentation?  Eg., "convert remove leading '--' and convert remaining '-' to '_'"